kale/embed/video_cnn/res3d.py

```diff
@@ -0,0 +1,335 @@
+"""
+Define C3D, MC3_18, Res3D_18, Res2plus1D_18 on Action Recognition from https://arxiv.org/abs/1711.11248
+Created by Xianyuan Liu from modifying https://github.com/pytorch/vision/blob/master/torchvision/models/video/resnet.py
+"""
```

Comment:
Check whether those w/o docstrings are obvious enough (e.g. for our new MSc who will work on AR). If not, add some docstrings.